Documentation/Installation/TutorialDdev.rst

```diff
@@ -0,0 +1,181 @@
+
+..  code-block:: bash
+
+    mkdir t3example
+    cd t3example
+
+Create a new DDEV Project
+-------------------------
+
+The `ddev config` command will prompt for information about your project. TYPO3 is in the list
+of preconfigured projects.
+
+..  code-block:: bash
+
+    ddev config
+
+    # Give the following answers when prompted:
+
+    Project name (t3example):
+
+    Docroot Location (current directory): public
+
+    Create docroot at /home/myuser/projects/t3/t3example/public? [Y/n] (yes): Y
+
+    Project Type [php, typo3, ...] (php): typo3
+
+project-type
+    Should always be "typo3"
+
+docroot
+    Is the folder in which all files that have to be reached by
+    the browser. This folder is commonly called :file:`public`.
+
+create-docroot
+    As the directory does not exist yet, this allows DDEV to create it for you.
+
+Alternatively you can skip the prompt by supplying all of the required parameters in a single command:
+
+..  code-block:: bash
+
+    ddev config  --project-type=typo3 --docroot=public --create-docroot
+
+Start the project
+-----------------
+
+..  code-block:: bash
+
+    ddev start
+
+The webserver is now running but TYPO3 is not installed.
+
+Install TYPO3
+-------------
+
+..  code-block:: bash
+
+    ddev composer create "typo3/cms-base-distribution:^11"
+
+As we just created the project and have no, answer yes
+when prompted if it is ok to overwrite files in this directory.
+
+You now have a **Composer-based TYPO3 installation**.
+
+Run the Installation Setup Tool
+-------------------------------
+
+Via the Console
+~~~~~~~~~~~~~~~
+
+..  code-block:: bash
+
+    ddev typo3cms install:setup
+
+When prompted enter a username and password for you main admin user
+(they will also be a :ref:`system maintainer <system-maintainer>`).
+
+Enter a name for the site that will be displayed in the backend.
+
+You can set up a basic site :bash:`site` displaying "Hello World"
+or have an empty installation by selecting :bash:`no`.
+
+Via the GUI:
+~~~~~~~~~~~~
+
+Create a file called :file:`FIRST_INSTALL` in your webroot
+
+..  code-block:: bash
+
+    ddev exec touch public/FIRST_INSTALL
+
+Open the installer
+
+..  code-block:: bash
+
+    ddev launch typo3/install.php
+
+Go to the TYPO3 backend:
+
+..  code-block:: bash
+
+    ddev launch typo3
+
+And login with the credentials you just provided.
+
+
+Managing the Database
+---------------------
+
+Upon calling :bash:`ddev config` DDEV automatically created a database for
+you. DDEV also created a file called :file:`public/typo3conf/AdditionalConfiguration.php`
+in which it stored the database credentials for you.
+
+During the installation setup process TYPO3 created all the tables it needed. If you want to have a look at the database, you can run the following command:
+
+..  code-block:: bash
+
+    ddev launch -p
+
+Sending E-Mail
+--------------
+
+DDEV modified :file:`public/typo3conf/AdditionalConfiguration.php`
```

Comment:
Actually DDEV creates this file. If there would have been an AdditionalConfiguration already, DDEV would not have modified it